src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -258,7 +258,7 @@
             [disabled]="(hasContent$ | async) === false"
             data-cy="nextflowtowerLaunchWith"
```

Comment:
Left this intentionally; it's referenced from the smoke tests, so if we change this, it's a minor hassle to synchronize since the smoke tests for all envs from the develop branch by default.